cleanlab/filter.py

```diff
@@ -398,6 +412,38 @@ class 0, 1, ..., K-1.
     return label_issues_mask
 
 
+def _find_label_issues_multilabel(
+    labels,
+    pred_probs,
+    return_indices_ranked_by=None,
+    rank_by_kwargs={},
+    filter_by="prune_by_noise_rate",
+    multi_label=False,
+    frac_noise=1.0,
+    num_to_remove_per_class=None,
+    min_examples_per_class=1,
+    confident_joint=None,
+    n_jobs=None,
+    verbose=False,
+):
+    ranked_label_issues_list = []
+    for i in range(0, len(pred_probs)):
+        y_labels = labels[i]
+        pred_probabilitites = pred_probs[i]
+        if confident_joint is None:
+            conf = None
+        else:
+            conf = confident_joint[i]
+        ranks = find_label_issues(
+            y_labels.astype(np.int32),
+            pred_probabilitites,
+            return_indices_ranked_by="self_confidence",
```

Comment:
keep as original
if it's None, and not None
if it's None, it returns a different an array of all true and false values (union1D to np.or ) , read code